Frameworks/UIKit/UITableViewCell.mm

```diff
@@ -1517,16 +1523,13 @@ - (void)prepareForReuse {
     }
     _editSelectedViewVisible = FALSE;
 
-    /*
-    if ( _backgroundView != nil ) {
-    [_backgroundView removeFromSuperview];
-    _backgroundView = nil;
-    }
-    if ( _selectedbackgroundView != nil ) {
-    [_selectedbackgroundView removeFromSuperview];
-    _selectedbackgroundView = nil;
+    // for any subivew which isn't internal subview used for building default chrome, need to remove them before re-use
+    for (UIView* subview in [self subviews]) {
```

Comment:
reference platform has the same issue.

---
In reply to: [94888495](https://github.com/Microsoft/WinObjC/pull/1631#discussion_r94888495) [](ancestors = 94888495,94888445)